### PR TITLE
EXC-GI_Pset_ImprovedGroundCommon

### PR DESCRIPTION
**Pull request by larswik**
# Pset: Pset_ImprovedGroundCommon

## Description:

Common properties for improved ground.

| Property              | Datatype                 | Description                                      |
| --------------------- | ------------------------ | ------------------------------------------------ |
| TreatmentHoleCount    | IfcCountMeasure          | Number of boreholes used for treatment.          |
| AvgTreatmentDepth   | IfcPositiveLengthMeasure | Average depth of boreholes used for treatment.   |
| AvgTreatmentSpacing | IfcPositiveLengthMeasure | Average spacing of boreholes used for treatment. |
| AvgDepthToTreatment   | IfcPositiveLengthMeasure | Average depth to treatment.                      |
| AvgTreatmentThickness | IfcPositiveLengthMeasure | Average treatment thickness.                     |

### CODE REVIEW

**Comment by larswik** [comment]
@JontyDunn: First example of pull request from "your" excel. Please check!
@hessf-ilf

File: IFC4x3/Properties/a/AvgDepthOfTreatment_2ATszB7MD5rOdgrmtnn2wi/DocProperty.xml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AvgDepthOfTreatment_2ATszB7MD5rOdgrmtnn2wi" Name="AvgDepthOfTreatment" UniqueId="8a776f4b-1d63-45d5-89ea-d70df1c42eac" PrimaryDataType="IfcPositiveLengthMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/a/AvgDepthOfTreatment_2ATszB7MD5rOdgrmtnn2wi/DocProperty.xml]: Propose renaming to "AvgTreatmentDepth".

File: IFC4x3/Properties/a/AvgSpacingOfTreatment_3IwKbHD4H2mxQff$jGfKO3/DocProperty.xml [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AvgSpacingOfTreatment_3IwKbHD4H2mxQffjGfKO3" Name="AvgSpacingOfTreatment" UniqueId="d2e94951-3444-42c3-b6a9-a7fb50a54603" PrimaryDataType="IfcPositiveLengthMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/a/AvgSpacingOfTreatment_3IwKbHD4H2mxQff$jGfKO3/DocProperty.xml]: Propose renaming to "AvgTreatmentSpacing".

File: IFC4x3/Properties/n/NumberOfTreatmentHoles_3XN76C94L2pvK6fvVtToP7/DocProperty.xml [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="NumberOfTreatmentHoles_3XN76C94L2pvK6fvVtToP7" Name="NumberOfTreatmentHoles" UniqueId="e15c718c-2445-42cf-9506-a797f7772647" PrimaryDataType="IfcCountMeasure" />
```
Comment (SergejMuhic) [inline IFC4x3/Properties/n/NumberOfTreatmentHoles_3XN76C94L2pvK6fvVtToP7/DocProperty.xml]: Propose renaming to "TreatmentHoleCount".

File: IFC4x3/Properties/a/AvgDepthOfTreatment_2ATszB7MD5rOdgrmtnn2wi/DocProperty.xml (hunk 1 above)
Comment (larswik) [reply IFC4x3/Properties/a/AvgDepthOfTreatment_2ATszB7MD5rOdgrmtnn2wi/DocProperty.xml]: Fixed in [fa34e3f](https://github.com/bSI-InfraRoom/IFC-Specification/pull/676/commits/fa34e3f56f97c85fda90a91839411efbd6712470)

File: IFC4x3/Properties/a/AvgSpacingOfTreatment_3IwKbHD4H2mxQff$jGfKO3/DocProperty.xml (hunk 2 above)
Comment (larswik) [reply IFC4x3/Properties/a/AvgSpacingOfTreatment_3IwKbHD4H2mxQff$jGfKO3/DocProperty.xml]: Fixed in [fa34e3f](https://github.com/bSI-InfraRoom/IFC-Specification/pull/676/commits/fa34e3f56f97c85fda90a91839411efbd6712470)

File: IFC4x3/Properties/n/NumberOfTreatmentHoles_3XN76C94L2pvK6fvVtToP7/DocProperty.xml (hunk 3 above)
Comment (larswik) [reply IFC4x3/Properties/n/NumberOfTreatmentHoles_3XN76C94L2pvK6fvVtToP7/DocProperty.xml]: Fixed in [fa34e3f](https://github.com/bSI-InfraRoom/IFC-Specification/pull/676/commits/fa34e3f56f97c85fda90a91839411efbd6712470)

**Pull request merged**